src/main/java/mate/controller/GetAllCarsController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars")
+public class GetAllCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        CarService carService =
+                (CarService) injector.getInstance(CarService.class);
+        req.setAttribute("cars", carService.getAll());
+        req.getRequestDispatcher("/WEB-INF/views/car/all.jsp").forward(req, resp);
```

Comment:
```suggestion
    private static final Injector injector = Injector.getInstance("mate");
    private final CarService carService =
                (CarService) injector.getInstance(CarService.class);

    @Override
    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
            throws ServletException, IOException {
        req.setAttribute("cars", carService.getAll());
        req.getRequestDispatcher("/WEB-INF/views/car/all.jsp").forward(req, resp);
```
initialize as a class-level field - fix all controllers